From 0067a7ed1e1c2177fc6782f790debc9a011f18a2 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Mon, 25 Jun 2012 10:37:12 +0200
Subject: [PATCH] remove useless extern declaration

This fixes segfault in checkdll libbf_xolo.so during build of
libreoffice 3.6 package for Fedora Rawhide. Do not ask me why...

Change-Id: I367a85ba2a10bc60bcd74ca90c2fd24e86f70967
---
 binfilter/inc/bf_xmloff/XMLChartPropertySetMapper.hxx |    2 --
 1 file changed, 2 deletions(-)

diff --git a/binfilter/inc/bf_xmloff/XMLChartPropertySetMapper.hxx b/binfilter/inc/bf_xmloff/XMLChartPropertySetMapper.hxx
index e4c9d27..a1e8932 100644
--- a/binfilter/inc/bf_xmloff/XMLChartPropertySetMapper.hxx
+++ b/binfilter/inc/bf_xmloff/XMLChartPropertySetMapper.hxx
@@ -37,8 +37,6 @@
 namespace rtl { class OUString; }
 namespace binfilter {
 
-extern const XMLPropertyMapEntry aXMLChartPropMap[];
-
 class SvXMLExport;
 
 // ----------------------------------------
-- 
1.7.10.2

